package.json

```
@@ -18,7 +18,7 @@
 		"eject": "react-scripts eject",
 		"lint": "eslint ./src/**/*.{ts,tsx}",
 		"lint:fix": "eslint --fix ./src/**/*.{ts,tsx}",
-		"prettier": "prettier --write ./src/**/*.{ts,tsx}"
+		"prettier": "prettier --write ./src/**/*.{ts,tsx,css,html,json}"
```

```suggestion
		"prettier": "prettier --write --ignore-unknown ."
```

prettier가 적용가능한 모든 파일에 적용시켜버리는 방법도 있습니다 ㅎㅎ..

---

!! 그런 방법도 있군요..ㅎㅎ
아직 이쪽(package.json) 스크립트를 작성하는 방법이나 prettier, lint 에서 제공하는 명령어를 잘 몰라서 그냥 쉽게쉽게 넘어갔는데, 시간이 되면 이 쪽도 좀 찾아볼 필요가 있겠어요 : )